src/components/DetailsInfo/detailsInfo.scss

```diff
@@ -168,5 +169,5 @@ $itemInfoWithoutPadding: item-info-without-padding;
 }
 
 :export {
-  itemInfoWithoutPadding: $itemInfoWithoutPadding;
+  iteminfowithoutpadding: $itemInfoWithoutPadding;
```

Comment:
why did you rename variable?

Reply:
It happens automatically when I use Prettier

Reply:
change it back please